src/api/include/migraphx/migraphx.hpp

```diff
@@ -1297,6 +1296,8 @@ inline void save(const program& p, const char* filename)
     call(&migraphx_save, p.get_handle_ptr(), filename, migraphx::file_options{}.get_handle_ptr());
 }
 
+#ifdef HAVE_ONNX
```

Comment:
These macros need to be defined in the migraphx.h header.

Reply:
I see, you added a config.h header, for the macros. I would like to see these macros inverted though(something like `#ifndef MIGRAPHX_MISSING_ONNX`)